Road/notebooks/data_cleaning.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    from source.features import data_cleaning as DC
</pre><p>Could you please use representative name instead of DC?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mawada-sweis/Road-Status/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I think its clear. do you recommend any names?</p>

Reply:
You could keep it `data_cleaning`

Reply:
Yes we will use it with its actual name, please update it